### ISSUE TITLE
Turn off validation for a specific file type

### ISSUE THREAD

**Issue by swernerx** [shinnn/vscode-stylelint#221]
I was kind of positively surprised that the extension report issues when using inline styles in JSX. Unfortunately it does not work well, so immediately after the first hurray, I am looking for ways to disable or fix it.

Code which is checked:

```js
<LeafletMap
  zoom={12}
  style={{ width: 300, height: 250 }}
  lat={latitude}
  long={longitude}
/>
```

Errors reported:

 88:51  ✖  Expected a trailing semicolon              declaration-block-trailing-semicolon
 88:53  ✖  Unexpected missing end-of-source newline   no-missing-end-of-source-newline

I think the main issue is with the postcss processing here, but the extension should somehow make it possible to limit the files which are checked e.g. extensions.

**Comment by shinnn** [comment]
Exact duplicate of https://github.com/shinnn/vscode-stylelint/issues/222
